lib/ide/src/controller/text.rs

```
@@ -107,7 +107,7 @@ impl Handle {
                 },
                 FileHandle::Module {controller} => {
                     controller.check_code_sync(content)?;
-                    controller.save_file().await?
```

Not intentional?
We shouldn't ignore error here.